spec/jobs/scheduled/auto_join_users_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Jobs::AutoJoinUsers do
+  it 'works' do
+    Jobs.run_immediately!
+    channel = Fabricate(:chat_channel, auto_join_users: true)
+    user = Fabricate(:user, last_seen_at: 1.minute.ago, active: true)
+
+    subject.execute({})
+
+    membership = UserChatChannelMembership.find_by(user: user, chat_channel: channel)
```

Comment:
would probably be interesting to track than the job is actually changing this value, ATM with this test following could be true before execute and nothing would fail